baseplate/__init__.py

```diff
@@ -55,6 +55,8 @@ def tracing_client_from_config(raw_config, log_if_unconfigured=True):
         The name for the service this observer is registered to.
     ``tracing.endpoint`` (optional)
```

Comment:
Should the in-app mode be marked as deprecated here? Or just flat out removed from docs but left functional?